docs/ruleset_functions/expected_system_type_from_Table_G3_1_1.md

```diff
@@ -0,0 +1,117 @@
+**Schema Version:** 0.0.22  
+
+**Description:** returns the expected system type based on the BAT, the number of floors of the building and the area of the BAT, and climate zone
+
+**Inputs:**
+- **building_area_type** - ("RESIDENTIAL", "PUBLIC_ASSEMBLY", "RETAIL", "HOSPITAL", "OTHER_NON_RESIDENTIAL", "HEATED-ONLY_STORAGE")
+- **number_of_floors** - this is the number of floors in the building integer greater than 0
+- **area_of_BAT** - this is the total area of the building area type
+
+**Returns:**  
+- **result**: a dict indicating the expected system type from table G3_1_1 ("SYS-1", "SYS-2", etc) and a string that indicates how the system was chosen, for example: {"EXPECTED_SYSTEM_TYPE": "SYS-4", "SYSTEM_ORIGIN": "PUBLIC_ASSEMBLY CZ_0_to_3a < 120,000 ft2"} - does not take into account G3.1.1 b-g
+ 
+**Function Call:**
+- **is_CZ_0_to_3a**
+
+## Logic:
+- create area_of_BAT_string, which will be appended to the details_of_system_selection string: `area_of_BAT_string = ""`
+- create number_of_floors_string, which will be appended to the details_of_system_selection string: `number_of_floors_string = ""`
+
+- get the climate zone category using the functions is_CZ_0_to_3a: `if is_CZ_0_to_3a():`
+	- set climate_zone_category to CZ_0_to_3a: `climate_zone_category = " CZ_0_to_3a"
+
+- else (the climate zone is 3b, 3c or 4 to 8): `else:`
+	- set climate_zone_category to CZ_3b_3c_or_4_to_8: `climate_zone_category = " CZ_3b_3c_or_4_to_8"
+
+- check if the building area type is residential: `if building_area_type == "RESIDENTIAL":`
+	- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+		- the expected system is System 1: `system_type = "SYS-2"`
+	- otherwise, it's  CZ_3b_3c_or_4_to_8: `else:`
+		- the expected system is System 2: `system_type = "SYS-1"
+
+
+- check if the building area type is Public Assembly: `if building_area_type == "PUBLIC_ASSEMBLY":`
+	- check if the space area is less than 120,000 ft2: `if area_of_BAT < 120000:`
+		- set area_of_BAT_string to "< 120,000": `area_of_BAT_string = " < 120,000 ft2"`
+		- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+			- the expected system is System 3: `system_type = "SYS-4"`
+		- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
+			- the expected system is System 4: `system_type = "SYS-3"
+	- otherwise, the space area is greater than 120,000 ft2: `else:`
+		- set area_of_BAT_string to ">= 120,000": `area_of_BAT_string = " >= 120,000 ft2"`
+		- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+			- the expected system is System 12: `system_type = "SYS-13"`
+		- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
+			- the expected system is System 13: `system_type = "SYS-12"
+
+
+- check if the building area type is heated-only storage: `if building_area_type == "HEATED-ONLY_STORAGE":`
+	- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+		- the expected system is System 9: `system_type = "SYS-10"`
+	- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
```

Comment:
explanation text needs to be corrected.
need to cycle through building segment to get the HVAC building area type. Incorrect data element being used for building area type  (see comment above which provides the correct data element)

Reply:
explanation text corrected

@supriyagoel - in this function `building_area_type` is passed into the function.  It is calculated using the function [get_HVAC_building_area_types_and_zones](https://github.com/pnnl/ruleset-checking-tool/blob/RDS/KJW/function_get_HVAC_building_area_types_and_zones/docs/ruleset_functions/get_HVAC_building_area_types_and_zones.md)

Reply:
updated